lib/Migrator/FilesMigrator.php

```diff
@@ -83,6 +85,16 @@ public function export(
 			throw new UserMigrationException("Could not copy files.");
 		}
 
+		try {
+			$versionsFolder = $this->root->get('/'.$uid.'/'.FilesVersionsStorage::VERSIONS_ROOT);
+			$output->writeln("Copying file versions…");
```

Comment:
```suggestion
			$output->writeln("Exporting file versions…");
```
Like the import message: "Importing file versions…"